src/config.rs

```diff
@@ -852,16 +855,6 @@ impl DbConfig {
         opts.set_max_log_file_size(self.info_log_max_size.0);
         opts.set_log_file_time_to_roll(self.info_log_roll_time.as_secs());
         opts.set_keep_log_file_num(self.info_log_keep_log_file_num);
-        if !self.info_log_dir.is_empty() {
```

Comment:
if `info_log_dir` is empty, default to `data-dir`(config value) + "/" + `storage::config::DEFAULT_ROCKSDB_SUB_DIR`

for raftdb, `info_log_dir` should default to `raftstore.raftdb-path`(config value).

Reply:
Done